app/presenters/tree_node/middleware_server.rb

```diff
@@ -0,0 +1,5 @@
+module TreeNode
+  class MiddlewareServer < Node
+    set_attribute(:title) { [@object.name.presence, @object.host_name].join(" - ") }
```

Comment:
Ok, it comes from (now dead `choose_node_identifier`) ... except that one uses `hostname`, not `host_name` ;)